scripts/us_census/acs5yr/subject_tables/subject_table_test.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Generic test for subject tables"""
+
+# pylint: disable=missing-module-docstring
+# pylint: disable=missing-class-docstring
+# pylint: disable=missing-function-docstring
+
+import os
+import sys
+import tempfile
+import unittest
+import subprocess
+
+# Allows the unittest to access table directories in relative path
+_SCRIPT_PATH = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.join(_SCRIPT_PATH, '.'))
+
+# These directories are excluded from testing
+_EXCLUDE_DIRS = ['common', 's2201', '__pycache__']
+
+
+def get_paths(table_dir):
+    """Gets paths to the spec, CSV, StatVar MCF, Column Map and test zip file.
+    Args:
+        table_dir: Path to the subject table directory.
+          Expected directory structure is (<code> refers to subject table code)
+          table_dir
+            - <code>_spec.json
+            - testdata
+                - <code>_output.mcf
+                - <code>_cleaned.csv
+                - column_map.json
+                - test zip file
+
+    Returns:
+        A tuple of the paths to the spec, CSV, Stat Var MCF, column map and test
+        zip file.
+    """
+    spec_path = None
+    statvar_mcf_path = None
+    csv_path = None
+    column_map_path = None
+    zip_path = None
+
+    # Find spec, Expects spec to be in table_dir
+    for filename in os.listdir(table_dir):
+        if '_spec.json' in filename:
+            spec_path = os.path.join(table_dir, filename)
+
+    # Expects StatVar MCF, Column Map, CSV and test zip to be in testdata folder
+    testdata_path = os.path.join(table_dir, 'testdata')
+    for filename in os.listdir(testdata_path):
+        if '_cleaned.csv' in filename:
```

Comment:
if 'cleaned.csv' in filename:
 will allow *cleaned.csv including ones without table names.

similarly below for other files to look without the '_'.

Reply:
Done.